pandas/tests/io/json/test_ujson.py

```diff
@@ -1,7 +1,10 @@
 try:
```

Comment:
Can you get rid of the try...except altogether? This looks like legacy code - `import json` should work dating back to at least Python 2.6

Reply:
sure, will do it

Reply:
@WillAyd I have done the changes, please review